### PR TITLE
[infra] Add onecc-docker directory to code format items

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit adds onecc-docker directory to code format items.

ONE-DCO-1.0.-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

related comment: https://github.com/Samsung/ONE/pull/9737#issuecomment-1250719414

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

File: infra/command/format [hunk 1]
```diff
@@ -157,6 +157,8 @@ function check_python_files() {
   FILES_TO_CHECK_PYTHON=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '\.py$'`)
   # Exceptional case: one-cmds don't have '.py' extension: ignore non-python source (cmake, etc) and ignore shell script: one-prepare-venv
   FILES_TO_CHECK_PYTHON+=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '^compiler/one-cmds/[^(\./)]*$' | egrep -v '^compiler/one-cmds/one-prepare-venv$'`)
+  # Exceptional case: onecc-docker don't have '.py' extension.
+  FILES_TO_CHECK_PYTHON+=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '^compiler/onecc-docker/[^(\./)]*$'`)
```
Comment (mhs4670go) [inline infra/command/format:161]: Maybe `Dockerfile` should be ignored?

Comment (jyoungyun) [reply infra/command/format:161]: `Dockerfile` is automatically ignored. Should I add it explicitly?

Comment (jyoungyun) [reply infra/command/format]: I explicitly added the `onecc-docker` script so that other files are not included.

File: infra/command/format [hunk 2]
```diff
@@ -157,6 +157,8 @@ function check_python_files() {
   FILES_TO_CHECK_PYTHON=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '\.py$'`)
   # Exceptional case: one-cmds don't have '.py' extension: ignore non-python source (cmake, etc) and ignore shell script: one-prepare-venv
   FILES_TO_CHECK_PYTHON+=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '^compiler/one-cmds/[^(\./)]*$' | egrep -v '^compiler/one-cmds/one-prepare-venv$'`)
+  # Exceptional case: onecc-docker don't have '.py' extension.
+  FILES_TO_CHECK_PYTHON+=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '^compiler/onecc-docker/onecc-docker'`)
```
Comment (mhs4670go) [inline infra/command/format:161]: (trivial)
```suggestion
  FILES_TO_CHECK_PYTHON+=(`echo "$FILES_TO_CHECK" | tr ' ' '\n' | egrep '^compiler/onecc-docker/onecc-docker$'`)
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**